vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -945,6 +945,10 @@ void WindowsTextInputComponentView::onFocusGained() noexcept {
     LRESULT lresult;
     DrawBlock db(*this);
     m_textServices->TxSendMessage(WM_SETFOCUS, 0, 0, &lresult);
+
+    if (m_props->clearTextOnFocus) {
```

Comment:
Right now, this is a build error. Not sure how I didn't catch this before.

Found the right way to go, will update in next iteration.